src/middlewared/middlewared/alert/service/telegram.py

```diff
@@ -0,0 +1,32 @@
+import json
+import requests
+
+from middlewared.alert.base import ThreadedAlertService
+from middlewared.schema import Dict, Str
+
+
+class TelegramAlertService(ThreadedAlertService):
+    title = "Telegram"
+
+    schema = Dict(
+        "telegram_attributes",
+        Str("bot_token", required=True, empty=False),
+        Str("chat_ids", required=True, empty=False),
+        strict=True,
+    )
+
+    def send_sync(self, alerts, gone_alerts, new_alerts):
+        token = self.attributes["bot_token"]
+        chatIds = [cId.strip() for cId in self.attributes["chat_ids"].split(",")]
```

Comment:
Can we use PEP8-compliant names? (i.e. `chat_ids`)

Also can we please try to be more friendly and accept more separators (i.e. simply whitespaces)
```
[chat_id.strip() for chat_id in self.attributes["chat_ids"].replace(",", " ").replace(";", " ").split() if chat_id.strip()]
```